Xamarin.Forms.Platform.iOS/Renderers/EntryRenderer.cs

```diff
@@ -249,5 +252,11 @@ void UpdateText()
 			if (Control.Text != Element.Text)
 				Control.Text = Element.Text;
 		}
-	}
+
+        bool ShouldChangeCharacters(UITextField textField, NSRange range, string replacementString)
+        {
+            var newLength = textField.Text.Length + replacementString.Length - range.Length;
```

Comment:
Null guards on textField or textField.Text? I ask because of the guards in https://stackoverflow.com/a/25224331